app.rb

```diff
@@ -1,13 +1,56 @@
 require "sinatra/base"
 require "sinatra/json"
 require "rack/cors"
+require "date"
+require "json"
+require "./spotify_api"
+require "./spotify_api_token"
+require 'digest/sha2'
 
 require "./db/setup"
 require "./lib/all"
+require 'pony'
+require 'rollbar'
+require 'rollbar/middleware/sinatra'
+
+require 'securerandom'
+
+if ENV['ROLLBAR_ACCESS_TOKEN']
+  Rollbar.configure do |config|
+    config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
+  end
+end
 
 class MyApp < Sinatra::Base
+  enable :sessions
+  set :session_secret, "hunter2"
```

Comment:
This needs to actually be a secret set in the environment. With this public, all of our logins are insecure.